cl/corpus_importer/management/commands/make_aws_manifest_files.py

```diff
@@ -587,6 +592,93 @@ def upload_list_of_records_for_users(
         )
 
 
+def get_account_id() -> str:
+    """
+    Retrieves the AWS account ID of the current execution role.
+
+    :return: The AWS account ID.
+    """
+    sts_client = boto3.client("sts")
+    return sts_client.get_caller_identity().get("Account")
+
+
+def get_lambda_arns() -> dict[str, str]:
+    """
+    Retrieves a dictionary of AWS Lambda function names and their ARNs.
+
+    :return: A dictionary where keys are Lambda function names (str) and values
+        are their corresponding ARNs (str).
+    """
+    lambda_client = boto3.client("lambda", region_name="us-west-2")
+    request = lambda_client.list_functions()
```

Comment:
According to documentation: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda/client/list_functions.html#Lambda.Client.list_functions

`list_functions` returns up to 50 functions. Is that enough for our use case, or could we miss a Lambda function? Otherwise, we might need some kind of pagination?